collectors/python.d.plugin/slurm/README.md

```diff
@@ -0,0 +1,23 @@
+<!--
+title: "Slurm queue monitoring with Netdata"
+custom_edit_url: https://github.com/netdata/netdata/edit/master/collectors/python.d.plugin/slurm/README.md
+sidebar_label: "Slurm Queue"
+-->
+
+# Slurm queue monitoring with Netdata
+
+Monitors slurm queue statistics using the squeue tool.  
+
+Execute `squeue` to grab information from the slurm queue.
```

Comment:
@mberr In which context do you need to execute squeue? 
I assume the collector does this for you, right?

Reply:
Yes, exactly.